modules/test/playwright/tests/osb-faro-web/segments.spec.ts

```diff
@@ -362,3 +368,264 @@ test('Search the Segment Profile Distribution', async ({apiHelpers, page}) => {
 		);
 	});
 });
+
+test('Segment Composition shows Active and Known individuals', async ({
+	apiHelpers,
+	page,
+}) => {
+	const anonymousIdentityID = '87';
+	const date1 = new Date();
+	const pageName = 'Liferay - AC Page';
+	const knownIndividualName = 'ac';
+	const dynamicSegmentName = 'Test Dynamic Segment';
+	const staticSegmentName = 'Test Static Segment';
+	const channelName = 'My Property - ' + getRandomString();
```

Comment:
Hey @ViniLopes20, can you please declare the variables right before they have been used?